pkg/network/state.go

```diff
@@ -320,17 +326,11 @@ func (ns *networkState) mergeConnections(id string, active map[string]*Connectio
 				closedConn.MonotonicSentBytes += activeConn.MonotonicSentBytes
 				closedConn.MonotonicRecvBytes += activeConn.MonotonicRecvBytes
 				closedConn.MonotonicRetransmits += activeConn.MonotonicRetransmits
+				closedConn.MonotonicTCPEstablished += activeConn.MonotonicTCPEstablished
+				closedConn.MonotonicTCPClosed += activeConn.MonotonicTCPClosed
 
 				ns.createStatsForKey(client, key)
 				ns.updateConnWithStatWithActiveConn(client, key, *activeConn, &closedConn)
-
-				// We also update the counters to reflect only the active connection
-				// The monotonic counters will be the sum of all connections that cross our interval start + finish.
-				if stats, ok := client.stats[key]; ok {
-					stats.totalRetransmits = activeConn.MonotonicRetransmits
-					stats.totalSent = activeConn.MonotonicSentBytes
-					stats.totalRecv = activeConn.MonotonicRecvBytes
-				}
```

Comment:
Are we sure about removing this? Did any tests break?

Reply:
When I was going through the code, this section looked like a duplicate to me.
None of the tests failed after I removed this, but I'll double check it just be sure 😬

Reply:
I think even if the tests don't break, let's not remove this unless we're making a commitment to refactor the `state.go` code in this PR, at which point we'll want to understand why it was added in the first place.

Reply:
We should however, add a task to the backlog to clean this up :)

Reply:
Sounds good to me, I'll revert that 👍